### PR TITLE
cmake:migrate apps CMakeLists for fsutils

### PR DESCRIPTION
**Pull request by xuxin930**
## Summary
migrate cmake build 
- inifile
- inih
- ipcfg
- libtinycbor
- mkgpt
- mkmbr
## Testing
CMake build eanble CONFIG_FSUTILS_INIFILE, CONFIG_FSUTILS_INIH,CONFIG_FSUTILS_IPCFG,CONFIG_FSUTILS_TINYCBOR_LIB,CONFIG_FSUTILS_MKGPT,CONFIG_FSUTILS_MKMBR

### CODE REVIEW

**Review by xiaoxiang781216 — APPROVED** [review]

File: fsutils/inih/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# the License.
+#
+# ##############################################################################
+
+if(CONFIG_FSUTILS_INIH)
+
+  # ############################################################################
+  # Config and Fetch inih lib
+  # ############################################################################
+  set(INIH_URL https://github.com/benhoyt/inih/archive/r42.tar.gz)
```
Comment (anchao) [inline fsutils/inih/CMakeLists.txt:26]: move after line 30

File: fsutils/libtinycbor/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#
+# ##############################################################################
+
+if(CONFIG_FSUTILS_TINYCBOR_LIB)
+
+  # ############################################################################
+  # Config and Fetch libtinycbor
+  # ############################################################################
+
+  set(TINYCBOR_VERSION 3cba6b11aaa0f6f674cd56ebaa573c4b65f71ee7)
```
Comment (anchao) [inline fsutils/libtinycbor/CMakeLists.txt:27]: move after line 34

**Review by jerpelea — APPROVED** [review]

File: fsutils/inih/CMakeLists.txt (hunk 1 above)
Comment (xuxin930) [reply fsutils/inih/CMakeLists.txt]: Done

File: fsutils/libtinycbor/CMakeLists.txt (hunk 2 above)
Comment (xuxin930) [reply fsutils/libtinycbor/CMakeLists.txt]: Done

**Pull request merged**